examples/organization-org_compliance/README.md

```diff
@@ -16,10 +16,15 @@ This example deploys Secure for Cloud into a GCP organizational account.
 ## Prerequisites
 
 1. Configure [Terraform **GCP** Provider](https://registry.terraform.io/providers/hashicorp/google/latest/docs)
-2. Following **roles** are required in your GCP organization/project credentials
+2. Run the script at `resources/fetch-gcp-projects.sh`. copy the output and provide it as input in the module
```

Comment:
Maybe we can say :
Run the script at `resources/fetch-gcp-projects.sh <orgID>` ?

Reply:
Done. Also added link to documentation